src/wasm-linker.cpp

```diff
@@ -381,12 +381,14 @@ void Linker::makeDynCallThunks() {
   }
 }
 
-Function* Linker::generateImportThunk(Name name, const FunctionType* funcType) {
+Function* Linker::getImportThunk(Name name, const FunctionType* funcType) {
+  Name thunkName = std::string("__importThunk_") + name.c_str();
```

Comment:
The optimization wouldn't help correctness, it would just help code size. Is this PR not just addressing code size, did I miss something?